### PR TITLE
Hadoop FDW: platform support updates and new Power install section

### PR DESCRIPTION
**Pull request by drothery-edb**
## What Changed?

- Updated the platform support info 
- Reformatted the install topic and added new section for RHEL8 Power

### CODE REVIEW

File: product_docs/docs/hadoop_data_adapter/2.1.0/05_installing_the_hadoop_data_adapter.mdx [hunk 1]
```diff
@@ -8,71 +8,59 @@ The Hadoop Foreign Data Wrapper can be installed with an RPM package. During the
...
+You can install the Hadoop Foreign Data Wrapper on the following operating systems on x86-64:
 
--   [RHEL or CentOS 7 PPCLE](#rhel_centos7_PPCLE)
--   [RHEL 7](#rhel7)
 -   [RHEL 8](#rhel8)
--   [CentOS 7](#centos7)
+-   [RHEL 7](#rhel7)
 -   [CentOS 8](#centos8)
+-   [CentOS 7](#centos7)
+-   [buntu/Debian](#on-debian-or-ubuntu)
```
Comment (nidhibhammar) [inline product_docs/docs/hadoop_data_adapter/2.1.0/05_installing_the_hadoop_data_adapter.mdx:19]: First 'U' is missing in name of the ubuntu platform [buntu/Debian]

Comment (drothery-edb) [reply product_docs/docs/hadoop_data_adapter/2.1.0/05_installing_the_hadoop_data_adapter.mdx]: Jinx. We must have caught that error at the same time. Fixed.

File: product_docs/docs/hadoop_data_adapter/2.1.0/05_installing_the_hadoop_data_adapter.mdx [hunk 2]
```diff
@@ -8,71 +8,59 @@ The Hadoop Foreign Data Wrapper can be installed with an RPM package. During the
 
 <div id="using_an_rpm_package_to_install_the_data_adapter" class="registered_link"></div>
 
-## Installing the Hadoop Foreign Data Wrapper using an RPM Package
+## Installing the on Linux x86-64 (amd64)
```
Comment (nidhibhammar) [inline product_docs/docs/hadoop_data_adapter/2.1.0/05_installing_the_hadoop_data_adapter.mdx:11]: I think we can remove 'the'

Comment (drothery-edb) [reply product_docs/docs/hadoop_data_adapter/2.1.0/05_installing_the_hadoop_data_adapter.mdx]: Thanks. Fixed.

**Pull request merged**